src/styles/_zindex.scss

```diff
@@ -1,6 +1,10 @@
-// so that cursor is always cursor:grabbing
-$dnd-cursor-back: 100500;
-// ghost for drag and drop. has to be always above other elements
-$dnd-ghost: 100499;
-
-$sticky-toolbar: 2000; // more than has popup
+// z-index system values
+$z-index-map: (
+    'background': -1,
+    'default': 0,
+    'forefront': 1,
+    'img-settings-button': 2,
+    'table-view-button': 100,
+    'table-cell-button': 110,
+    'sticky-toolbar': 990,
```

Comment:
1. header sticky toolbar+
2. header sticky toolbar
3. context toolbar+ (popups, hints)
4. context toolbar
5. suggest popup+ (for hints), (`...` buttons menu)
6. suggest popup
7. block panel popup

context is popup from text selection
suggest is `/`, `:`

Reply:
if you need to change z-index in uikit component use style https://github.com/gravity-ui/uikit/blob/main/src/components/types.ts#L4C5-L4C27

like this:
```
style={{
   // ...otherStyles,
   zIndex: var(--md-editor-z-index-level-suggest);
}}
```